src/components/DashboardMenu.tsx

```diff
@@ -31,43 +31,43 @@ const DashboardMenu: React.FC = () => {
           <li>
             <TabButton onSelect={() => handleSelect("main-course")}>
               <img className="courses_icon" src={Main} alt="main course" />
-              Main course
+              <p className="courses_title">Main course</p>
```

Comment:
if all of your paragraphs will have the same changes/class, maybe you can just access directly the p under the parent classname?